lib/rails_workflow/dependency_resolver.rb

```diff
@@ -25,7 +25,7 @@ def build_new_operations(operation)
             process, operation_template, completed_dependencies
           ).create_operation
         end
-      end
+      end.compact
```

Comment:
Previously, this was returning `[nil]` in some cases, which was causing `build_new_operations` below to crash when it tried to concat new_operations.